src/Sylius/Bundle/ShopBundle/Resources/views/Product/_singleProduct.html.twig

```diff
@@ -0,0 +1,13 @@
+{% from 'SyliusShopBundle:Product:macros.html.twig' import sylius_display_price %}
+
+<div class="content">
+    <div class="header">{{ product.name|truncate(18) }}</div>
+</div>
+<div class="extra content">
+    <span class="right floated">
+        <button class="ui primary button">Add to cart</button>
+    </span>
+    <span>
+        {{ sylius_display_price(product.firstVariant) }}
```

Comment:
We should add `if (product.firstVariant is not null)` here, because we allow for products without variants.